__docs__/wonder-blocks-modal/flexible-dialog.stories.tsx

```diff
@@ -0,0 +1,539 @@
+/* eslint-disable no-alert */
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+import type {Meta, StoryObj} from "@storybook/react";
+
+import Button, {ActivityButton} from "@khanacademy/wonder-blocks-button";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
+import {BodyText, Heading} from "@khanacademy/wonder-blocks-typography";
+
+import {ModalLauncher, FlexibleDialog} from "@khanacademy/wonder-blocks-modal";
+import packageConfig from "../../packages/wonder-blocks-modal/package.json";
+
+import ComponentInfo from "../components/component-info";
+import FlexibleDialogArgTypes from "./flexible-dialog.argtypes";
+import {allModes} from "../../.storybook/modes";
+
+import celebrationBg from "../../static/celebration_bg.svg";
+import celebrationChest from "../../static/celebration-chest.svg";
+
+const customViewports = {
```

Comment:
I wonder if this is something we can standardize so we can reuse it for other components too!

Reply:
Yes, good call! I see that code is repeated in a bunch of the modal stories. I'd prefer to do that as a follow-up, so I created an issue here: https://khanacademy.atlassian.net/browse/WB-2053

Reply:
Thank you! ❤️